recipes/ormsgpack/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "ormsgpack" %}
+{% set version = "1.4.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/ormsgpack-{{ version }}.tar.gz
+  sha256: b0feb4b80b871df04bb5db79ff2ea1bee8a5483e70676387449631d778ad9b18
+
+build:
+  noarch: python
```

Comment:
This should be removed. This package is not noarch

Reply:
Silly mistake on my part; didn't check this from grayskull. Thanks!